lib/libspl/os/freebsd/getmntany.c

```diff
@@ -61,3 +61,16 @@ getextmntent(const char *path, struct extmnttab *entry, struct stat64 *statbuf)
 	statfs2mnttab(&sfs, (struct mnttab *)entry);
 	return (0);
 }
+
+int
+fgetextmentent(int fd, struct extmnttab *entry, struct stat64 *statbuf)
```

Comment:
```suggestion
fgetextmntent(int fd, struct extmnttab *entry, struct stat64 *statbuf __unused)
```

Reply:
Sigh. I must have pushed wrong branch. Thanks for pointing this out.